docs/reference/settings/monitoring-settings.asciidoc

```diff
@@ -49,7 +49,7 @@ and {ls} is ignored.
 [[xpack-monitoring-collection-interval]]
 // tag::monitoring-collection-interval-tag[]
 `xpack.monitoring.collection.interval` {ess-icon}::
-deprecated:[6.3.0,"Use `xpack.monitoring.collection.enabled` set to `false` instead."] 
+deprecated:[6.3.0,"Use `xpack.monitoring.collection.enabled` set to `false` instead."]
```

Comment:
sorry about the whitespace changes .... not sure why this file is special. Github diff can ignore these with a button on the top left, or this link: https://github.com/elastic/elasticsearch/pull/98513/files?w=1